be/src/vec/data_types/serde/data_type_serde.h

```diff
@@ -57,9 +59,20 @@ class IDataType;
 // the developer does not know how many datatypes has to deal.
 
 class DataTypeSerDe {
+public:
+    struct FormatOptions {
+        bool use_lib_format = false;
```

Comment:
`use_lib_format` could you add some example to explain more details of this flag?